index.js

```diff
@@ -241,14 +266,20 @@ function handlePaths(files, excludePatterns, globOptions) {
 
 				searchedParents[file] = true;
 
-				var pattern = path.join(file, '**', '*.js');
+				// var pattern = path.join(file, '**', '*.js');
+				var patternList = [];
+				globOptions.extensions.forEach(function (ext) {
+					patternList.push(path.join(file, '**', '*.' + ext));
```

Comment:
Inconsistency between https://github.com/avajs/ava-files/pull/9/files#diff-168726dbe96b3ce427e7fedce31bb0bcR187. `path.join()` is used here, unlike in that other place. I think it should be used in both places.